### ISSUE TITLE
UTF-8 in labels causes problems

### ISSUE THREAD

**Issue by PeterSurda** [mailchuck/PyBitmessage#180]
UTF-8 in labels (addressbooks, subscriptions, ...), like umlauts, does not display correctly. Also, it could be that if it is invalid UTF-8, the Bitmessage window does not show at all (one bug reporter), but I cannot reproduce that.
- [x] Editing a subscription in UI
- [ ] Editing a subscription in SQL
- [x] Editing an addressbook entry in UI
- [ ] Editing an addressbook entry in SQL
- [x] Editing an address label in UI
- [ ] Editing an address label in SQL

**Comment by Erkan-Yilmaz** [comment]
I tried and it seems I can reproduce it.
1. OK:

a. e.g. adding umlauts in the text of a message + subject works 

b. the umlaut is shown (also after receiving when I sent to myself this msg)

2.

but: when I add the umlaut in the address book as description (1) of the BM address: 

it discards the text and takes the original text (2) again

(1) e.g. "erkan Nö"
(2) "erkan"

nothing more done than above 2 points, so I can at least try those again when a fix would be available

using https://github.com/mailchuck/PyBitmessage v 0.5.7
with commit: 2f9501fa1aafb86cc8d6be1d041abfe1b30c225a

**Comment by PeterSurda** [comment]
That part I can reproduce too. But one guy reported that his PyBM does not start at all (the main QT thread crashes in rerenderSubscriptions) and that I can't reproduce.

**Comment by PeterSurda** [comment]
@Erkan-Yilmaz can you try now? The UI should be fixed, just left is checking what happens if you manually create invalid utf8 in keys.dat or the database.

**Comment by Erkan-Yilmaz** [comment]
@PeterSurda I have tested my use case above ( "erkan äöü")
OK: "erkan äöü" stays when I switch between tabs
BUT: when I restart bitmessage it is again only "erkan"

I am on commit c6b4237

**Comment by PeterSurda** [comment]
@Erkan-Yilmaz what OS?

**Comment by Erkan-Yilmaz** [comment]
@PeterSurda I am sending this info right now to you via BM's support mail

**Comment by PeterSurda** [comment]
I can't reproduce this (Windows and Linux). Are you editing an entry in the addressbook (rather than one of your own addresses)? The black ones in the "Send" tab?

**Comment by Erkan-Yilmaz** [comment]
yes, I was editing one of the "Address book" entries (1) from the Send tab (not the Address field which holds the BM-address)

(1) which is also called "Label" when one creates a new contact

**Comment by PeterSurda** [comment]
I'm sorry but I can't reproduce this. I'll postpone this until there is a more reliable way for me to reproduce.
